bundles/org.openhab.binding.fenecon/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.3.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.fenecon</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: FENECON Binding</name>
+
```

Comment:
Please add connection tag and if possible others too.

Reply:
Done

Reply:
The connection tag is missing and the inceptionYear/developers tags are not used for official openHAB bindings (they are only for marketplace/3rd party), please remove those.